### PR TITLE
Provide a better git commit template

### PR DESCRIPTION
**Pull request by nicholasjhenry**
This is to encourage team members to provide more context (i.e. the why) a change is being made.

### CODE REVIEW

File: gitmessage [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
 
-Co-authored-by: Hugo Frappier <14030+frahugo@users.noreply.github.com>
-Co-authored-by: Nicolas Charlery <bokay@users.noreply.github.com>
-Co-authored-by: Nicholas Henry <40253+nicholasjhenry@users.noreply.github.com>
-Co-authored-by: Tetiana Dushenkivska <7498434+tetiana12345678@users.noreply.github.com>
+# Explain why this change is being made
+# |<----   Try To Limit Each Line to a Maximum Of 72 Characters   ---->|
+
+# Provide links or keys to any relevant tickets, articles or other resources
+# Example: Github issue #23
```
Comment (frahugo) [inline gitmessage:9]: Do we want to always link to the GitHub issue? Maybe only if it is an external GitHub issue.

Comment (nicholasjhenry) [reply gitmessage:9]: @frahugo Correct, we don't want to always link to the ticket". I think that is the intent here is "Provide links or keys to any relevant tickets".

**Comment by nicholasjhenry** [comment]
I'm merging this in now and we can make updates as necessary.

**Comment by nicholasjhenry** [comment]
@team Please update your dotfiles as this will give you the latest `gitmessage` template. If you're not setup with `civilcode` dotfiles, please see:

https://github.com/civilcode/dotfiles

**Pull request merged**